infrastructure/conf/nginx-conf-deploy/default

```diff
@@ -69,16 +69,11 @@ server {
         fastcgi_param SCRIPT_FILENAME /home/site/wwwroot/api/public/index.php;
     }
 
-    # rewrite root-level possible lang prefix to talentsearch index.html
+    # rewrite root-level possible lang prefix to apps/web index.html
     location ~ "^(?:/[a-z]{2})?/?$" {
         add_header Cache-Control no-cache;
         expires 0;
-        rewrite "^(?:/[a-z]{2})?/?$" /frontend/talentsearch/dist/index.html break;
-    }
-
-    # find the admin config in the writable home root
-    location = /admin/config.js {
```

Comment:
Is this ok to remove since it's admin?

Reply:
definitely not, that was a bad mistake. Great catch!

Fixed in [re-add admin config to nginx deploy](https://github.com/GCTC-NTGC/gc-digital-talent/pull/5344/commits/8d2daa6e60c51dd85a2371caa1044138ad2fe951)